proto/galileo/faucet/v1/faucet.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package galileo.faucet.v1;
+
+import "penumbra/core/keys/v1/keys.proto";
+import "penumbra/core/txhash/v1/txhash.proto";
+
+// Runs a testnet faucet service.
+service FaucetService {
```

Comment:
> Runs a testnet faucet service.

This seems like a good spot to stress the non-authenticated nature of the RPC. But since we're not publishing these docs externally anywhere (e.g. BSR), fine as-is.